Gemfile

```diff
@@ -2,6 +2,13 @@ source 'https://rubygems.org'
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
-gem 'cql_qdm_patientapi', github: 'projecttacoma/cql_qdm_patientapi', branch: 'bonnie-v2.3'
+gem 'cql_qdm_patientapi', github: 'projecttacoma/cql_qdm_patientapi', branch: 'master'
+
+# below can be comented out to use default
+gem 'cqm-models', github: 'projecttacoma/cqm-models', branch: 'datatype_id_use_schema'
+
+group :test, :development do
```

Comment:
I made a comment about this same thing in the corresponding PR in cqm-models. This should be in the gemspec, if this additional dependency makes sense to have.